src/dlgTriggerEditor.cpp

```diff
@@ -3705,29 +3705,19 @@ void dlgTriggerEditor::addAction(bool isFolder)
         TAction* pParentAction = mpHost->getActionUnit()->getAction(parentID);
         if (pParentAction) {
             // insert new items as siblings unless the parent is a folder
-            if (!pParentAction->isFolder()) {
```

Comment:
:+1: for zapping the `goto`s - i just hope you have correctly untangled all the potential execution paths as that is the sort of thing that can get confusing quite quickly. :woozy_face:

Reply:
Yep, just double-checked that.